.github/CODEOWNERS

```
@@ -1388,6 +1388,9 @@ x-pack/test/security_solution_cypress/cypress/tasks/expandable_flyout  @elastic/
 /x-pack/plugins/stack_connectors/server/connector_types/sentinelone @elastic/security-defend-workflows
 /x-pack/plugins/stack_connectors/common/sentinelone @elastic/security-defend-workflows
 
+## Shared OAS schemas
+/x-pack/plugins/security_solution/common/api/common_schemas @elastic/security-detection-rule-management @elastic/security-detection-engine
```

```suggestion
/x-pack/plugins/security_solution/common/api/model @elastic/security-detection-rule-management @elastic/security-detection-engine
```
For each individual domain, shared components within the domain go into a `model` folder - I think we should use the same naming scheme at the top level of the `/api` folder as well. Shared components across the entire security solution API would be found in `/api/model`.

---

Agree. I hesitated between `model` and something less generic but unified naming is much better.